libraries/artifactory_gavc_downloader.rb

```diff
@@ -31,7 +32,7 @@ def find_newest_version_for_coordinates(group: raise('group is required'),
                                           artifact: raise('artifact is required'),
                                           version: raise('version is required'),
                                           repos: raise('repo(s) are required'))
-    return version unless wildcarded_version?(version)
+    return version if (!wildcarded_version?(version) && !snapshot_version?(version))
```

Comment:
Can you explain this change? It seems like this logic is no different than before, given a snapshot version such as '1.0.0-SNAPSHOT'. Snapshots don't have wildcards in them, as far as I know.